src/features/defi/providers/yearn/components/YearnManager/Withdraw/components/Status.tsx

```diff
@@ -44,19 +48,31 @@ export const Status = () => {
   const feeAsset = useAppSelector(state => selectAssetById(state, feeAssetId))
   const feeMarketData = useAppSelector(state => selectMarketDataById(state, feeAssetId))
 
-  const confirmedTransaction = useAppSelector(gs => selectTxById(gs, state?.txid || 'undefined'))
+  const web3 = getWeb3Instance()
 
-  useEffect(() => {
-    if (confirmedTransaction && confirmedTransaction.status !== 'pending' && dispatch) {
```

Comment:
@kaladinlight and i suspect this is to do with websocket stability. this selector should update when the tx comes in, and this logic is otherwise sound.

Reply:
This should be an easy one to test just to validate the logic and I can maybe cut over to the websocket reconnect logic as it is the root cause of a lot of these issues

Reply:
we're indexing this wrong - use https://github.com/shapeshift/web/blob/develop/src/state/slices/txHistorySlice/utils.ts#L50 to build up a tx index, not a raw id, and this selector will work.

while testing - ensure the websocket actually sends the pending/confirmed tx too as we've had sporadic issues w/ those

Reply:
@0xdef1cafe @kaladinlight Websocket was working, once the tx index was sent in properly it read it from state.

I've updated to build the tx index.